src/features/processPicker.ts

```diff
@@ -35,6 +37,80 @@ export class AttachPicker {
     }
 }
 
+export class RemoteAttachPicker {
+    public static ShowAttachEntries(args : any): Promise<string> {
+        // Grab selected name from UI
+        var name : string = args.name;
```

Comment:
Do we need to error out if `name` is undefined, not a string, or the empty string?